### PR TITLE
Add more MySQL images and MongoDB 3.4

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

File: mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# MySQL 5.6
+
+```Dockerfile
+FROM quay.io/continuouspipe/mysql5.6:v1.0
+```
+
+## How to build
+```bash
+./build.sh
+docker-compose build mysql56 mysql57 mysql80.
```
Comment (kierenevans) [inline mysql/README.md]: Extra dot which would cause this command to fail

File: mongodb/build.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+set -xe
+
+if [ -L "$0" ] ; then
+    DIR="$(dirname "$(readlink -f "$0")")" ;
+else
+    DIR="$(dirname "$0")" ;
+fi ;
+
+for IMAGE_TAG in 3.4; do
```
Comment (kierenevans) [inline mongodb/build.sh]: This is being complained about by shellcheck

Comment (andytson-inviqa) [inline mongodb/build.sh]: hmm, I made it so it'd be quick to add a new tag

File: mysql/README.md (hunk 1 above)
Comment (andytson-inviqa) [inline mysql/README.md]: this is fixed

**Pull request merged**